src/js/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -86,56 +191,136 @@ class RoutedSavableReviewPage extends React.Component {
+          <div>
+            {chapters.map(chapter => (
+              <ReviewCollapsibleChapter
+                expandedPages={chapter.expandedPages}
+                chapterFormConfig={chapter.formConfig}
+                chapterKey={chapter.name}
+                form={form}
+                formContext={formContext}
+                key={chapter.name}
+                onEdit={this.handleEdit}
+                open={chapter.open}
+                pageKeys={chapter.pageKeys}
+                setData={this.setData}
+                setValid={setValid}
+                showUnviewedPageWarning={chapter.showUnviewedPageWarning}
+                toggleButtonClicked={() => this.handleToggleChapter(chapter)}
+                uploadFile={this.props.uploadFile}
+                viewedPages={viewedPages}/>
+            ))}
+          </div>
+        </div>
+        <p><strong>Note:</strong> According to federal law, there are criminal penalties, including a fine and/or imprisonment for up to 5 years, for withholding information or for providing incorrect information. (See 18 U.S.C. 1001)</p>
+        <PrivacyAgreement
+          required
+          onChange={this.props.setPrivacyAgreement}
+          checked={form.data.privacyAgreementAccepted}
+          showError={form.submission.hasAttemptedSubmit}/>
+        <SubmitButtons
+          onBack={this.goBack}
+          onSubmit={this.handleSubmit}
+          submission={form.submission}
+          renderErrorMessage={this.renderErrorMessage}/>
+        {!disableSave && form.submission.status === 'error' ? null : <div>
+          <SaveStatus
+            isLoggedIn={user.login.currentlyLoggedIn}
+            showLoginModal={user.login.showModal}
+            toggleLoginModal={this.props.toggleLoginModal}
+            form={form}>
+          </SaveStatus>
+          <SaveFormLink
+            locationPathname={location.pathname}
+            form={form}
+            user={user}
+            saveAndRedirectToReturnUrl={this.props.saveAndRedirectToReturnUrl}
+            toggleLoginModal={this.props.toggleLoginModal}/>
+        </div>
+        }
       </div>
     );
-
-    return (
-      <ReviewPage
-        {...this.props}
-        setData={this.setData}
-        formContext={getFormContext({ user, form })}
-        contentAfterButtons={form.submission.status === 'error' ? null : contentAfterButtons}
-        renderErrorMessage={this.renderErrorMessage}/>
-    );
   }
 }
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
+  // from ownprops
+  const formConfig = ownProps.route.formConfig;
+  const pageList = ownProps.route.pageList;
+
+  // from redux state
+  const form = state.form;
+  const formData = state.form.data;
+  const openChapters = getReviewPageOpenChapters(state);
+  const user = state.user;
+  const viewedPages = getViewedPages(state);
+
+  const chapterNames = getActiveChapters(formConfig, formData);
+  const disableSave = formConfig.disableSave;
```

Comment:
Pulled this decision out of the router. There are only a couple of places where the behavior diverged between `ReviewPage` and `RoutedSavableReviewPage` and I thought it'd be easier to consolidate them into a single component.